service/templates/base.html

```diff
@@ -71,7 +71,7 @@
                     <span>Skip to main content</span>
                 </a>
                 <a class="logo" href="//jisc.ac.uk">
-                    <img alt="Jisc logo" src="{{url_for('static', filename='images/jisc-logo.png')}}">
+                     <img alt="Jisc logo" width="43" height="25" src="{{url_for('static', filename='images/jisc-logo.png')}}">
```

Comment:
Just a note: The same logo image is used in 2 places - footer and banner. It has to be resized at least for the banner. The previous image was w43xh25 (used by CL) the current one is w316xh184.